src/components/AdoptionForm.js

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from "react";
+import ErrorList from "./ErrorList";
+import validateForm from "../functions/validateForm";
+import postData from "../functions/postData";
+
+const AdoptionForm = props => {
+  const POST_API_PATH = "/api/v1/applicants";
+  const defaultApplicant = {
+    name: "",
+    phoneNumber: "",
+    email: "",
+    homeStatus: ""
+  };
+
+  const [newApplicant, setNewApplicant] = useState(defaultApplicant);
+  const [errors, setErrors] = useState({});
+
+  const clearForm = () => setNewApplicant(defaultApplicant);
+
+  const handleChange = e => {
```

Comment:
Though it doesn't break the code and is often a matter of personal preference, we would like you to use `event` instead of `e`. When we teach React we tell everyone to be explicit and not use 1 character sized variables or calls, this way everyone in your group knows what is being referenced when they see it.

Reply:
will do